app/models/ability.rb

```diff
@@ -51,12 +51,16 @@ def grant_permissions
       can_manage_items?(current_user.roles(new_apo_id)) && can?(:manage_item, cocina_object)
     end
 
-    can :view_content, Cocina::Models::DRO do |dro|
-      can_view? current_user.roles(dro.administrative.hasAdminPolicy)
+    can :view_content, Cocina::Models::DRO do |cocina_item|
+      can_view? current_user.roles(cocina_item.administrative.hasAdminPolicy)
     end
 
-    can :view_metadata, [Cocina::Models::Collection, Cocina::Models::DRO, Cocina::Models::AdminPolicy] do |dro|
-      can_view? current_user.roles(dro.administrative.hasAdminPolicy)
+    can :view_metadata, [Cocina::Models::Collection, Cocina::Models::DRO, Cocina::Models::AdminPolicy] do |cocina_object|
+      can_view? current_user.roles(cocina_object.administrative.hasAdminPolicy)
+    end
+
+    can :view_metadata, Cocina::Models::AdminPolicy do |cocina_admin_policy|
+      can_view? current_user.roles(cocina_admin_policy.externalIdentifier)
```

Comment:
This ability was restored from just prior to #2486.

Reply:
Should `Cocina::Models::AdminPolicy` be removed from line 58 too?

Reply:
Alternatively, if we still want/need to keep that behavior can we move it into this block so that there is only one block for `:view_metadata, Cocina::Models::AdminPolicy`?  I think that adds to readability.

Reply:
@jcoyne I don't think so? At least, we used to have this:

```ruby
  can :view_metadata, ActiveFedora::Base do |dor_item|
    can_view? current_user.roles(dor_item.admin_policy_object.pid) if dor_item.admin_policy_object
  end
```

Reply:
@jcoyne made that change. thanks!